spec/ios_l10n_helper_spec.rb

```diff
@@ -160,4 +160,82 @@ def file_encoding(path)
       end
     end
   end
+
+  describe '#download_glotpress_export_file' do
+    let(:gp_fake_url) { 'https://stub.glotpress.com/rspec-fake-project' }
+
+    describe 'request query parameters' do
+      it 'passes the expected params when no filters are provided' do
+        # Arrange
+        stub = stub_request(:get, "#{gp_fake_url}/fr/default/export-translations?format=strings").to_return(body: 'content')
+        dest = StringIO.new
+        # Act
+        described_class.download_glotpress_export_file(project_url: gp_fake_url, locale: 'fr', filters: nil, destination: dest)
+        # Assert
+        expect(stub).to have_been_made.once
+        expect(dest.string).to eq('content')
+      end
+
+      it 'passes the expected params when a list of filters is provided' do
+        # Arrange
+        stub = stub_request(:get, "#{gp_fake_url}/fr/default/export-translations?format=strings&filters%5Bstatus%5D=current&filters%5Bterm%5D=foobar").to_return(body: 'content')
+        dest = StringIO.new
+        # Act
+        described_class.download_glotpress_export_file(project_url: gp_fake_url, locale: 'fr', filters: { status: 'current', term: 'foobar' }, destination: dest)
+        # Assert
+        expect(stub).to have_been_made.once
+        expect(dest.string).to eq('content')
+      end
+    end
+
+    describe 'destination parameter' do
+      # it 'accepts an IO (File)' # Not really needed, as all previous tests already cover this by using a `StringIO`
+
+      it 'accepts a String (path)' do
+        Dir.mktmpdir('a8c-release-toolkit-tests-') do |tmp_dir|
+          # Arrange
+          # Note: in practice it seems that GlotPress's `.strings` exports are using UTF8 (but served as `application/octet-stream`)
+          #       but it does not hurt to ensure the download to a file can work with UTF16 ()and copy the binary stream verbatim)
```

Comment:
Other naming nitpick:
```suggestion
          #       but it does not hurt to ensure the download to a file can work with UTF-16 ()and copy the binary stream verbatim)
```